CONTRIBUTING.md

```diff
@@ -96,9 +97,98 @@ Ensure all the following conditions are met (on top of you agreeing with the cha
 
 To protect the Hyperledger Cactus source code, GitHub pull requests are accepted from forked repositories only. There are also quality standards identified and documented here that will be enhanced over time.
 
-
-
-
+### Create local branch
+
+> Whenever you begin work on a new feature or bugfix, it's important that you create a new branch. 
+
+<<<<<<< HEAD
+1. Clone your fork to your local machine
+2. Setup your local fork to keep up-to-date (optional)
+=======
+1. Clone your fork to your local machine.
+2. Setup your local fork to keep up-to-date (optional).
+>>>>>>> a054a8b... docs(contributing): create branch
+   ```
+   # Add 'upstream' repo to list of remotes
+   git remote add upstream https://github.com/hyperledger/cactus.git
+
+   # Verify the new remote named 'upstream'
+   git remote -v
+
+<<<<<<< HEAD
+   # Fetch from upstream remote
+   git fetch upstream
+
+   # View all branches, including those from upstream
+   git branch -va
+
+   # Checkout your master branch and merge upstream. 
+   # Run those commands whenever you want to synchronize with master branch
+   git checkout master
+   git merge upstream/master
+   ```
+3. Create your branch
+=======
+   # Checkout your master branch and rebase to upstream. 
+   # Run those commands whenever you want to synchronize with master branch
+   git fetch upstream
+   git checkout master
+   git rebase upstream/master
+   ```
+3. Create your branch.
+>>>>>>> a054a8b... docs(contributing): create branch
```

Comment:
@sfuji822 Thank you for updating to have `rebase` instead of `merge`!
It looks like maybe you had a merge conflict and then somehow the comparison view of that merge conflict got committed? I recognize this syntax from when I have merge conflicts in git

```
>>>>>>> a054a8b... docs(contributing): create branch
...
<<<<<<< HEAD
```

Reply:
Sorry. I did not noticed there are conflict during such short work.
I believe all conflicts are solved now.

Reply:
> Sorry. I did not noticed there are conflict during such short work.
> I believe all conflicts are solved now.

No problem, I combined the 4 separate commits into one and did a force push. I hope it's okay (If it is, then no further action necessary).
The content is 100% equivalent, just instead of the merging back and forth in multiple commits there is now one.
I did leave the original state of it on a backup branch as well, just in case.